server.js

```diff
@@ -651,20 +651,59 @@ proxyForAuditing.on('error', function(err, req, res) {
   serverUtils.serverError(JSON.stringify(err), req, res);
 });
 
-var couchDbVersionCheck = function(callback) {
-  db.getCouchDbVersion(function(err, version) {
-    console.log('CouchDB Version:', version);
+const nodeVersionCheck = callback => {
+  try {
+    console.log('Node Version:', process.version);
+
+    const version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
```

Comment:
Actually this `[1]` seems a bit dodgy - IIUC this will give you the major version number.  But below, you're trying to get sub-parts of `version`, which will give specific characters from the string matched on this line.

E.g.
```
> process.versions.node
'1.2.3'
> majorVersion = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
'1'
> majorVersion[0]
'1'
> majorVersion[1]
undefined
```

I assume that `const version` is meant to be assigned to the full version string (`1.2.3`).  If so, it might be nice to use it in the debug output above as well.  If not, it might be better called `majorVersion`.